java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -2715,7 +2710,7 @@ else if (action.get().getActionType().equals(ActionFactory.TYPE_KICKSTART_INITIA
                             KickstartAction ksAction = (KickstartAction) action.get();
                             if (!ksAction.getKickstartActionDetails().getUpgrade()) {
                                 // Delete salt key from master
-                                saltKeyUtils.deleteSaltKey(ksAction.getSchedulerUser(), minionId);
+                                saltApi.deleteKey(minionId);
```

Comment:
if i understand this correctly the previous version took the users permissions into account. Whats the reason for removing it?

Reply:
I thought it was unintentionally used `saltKeyUtils` instead of `saltApi` , since all the other methods in `SaltServerActionService` used the last one. But yes, you are totally right :) . I also checked if user permission was checked before  calling `handleAction` and they are not, so I am going to remove this change. Thanks for  pointing that out!